### ISSUE TITLE
Update to WebVR 1.1

### ISSUE THREAD

**Issue by spite** [spite/WebVR-Extension#9]
From the new spec, and this https://github.com/w3c/webvr/blob/gh-pages/js/webvr-1-1.js

**Comment by AVGP** [comment]
So that needs `VRFrameData` and the corresponding methods, right?

**Comment by spite** [comment]
Yes, right now it relies on the page using a polyfill (https://toji.github.io/webvr-samples/js/third-party/webvr-polyfill.js).

The blockers right now are https://github.com/spite/WebVR-Extension/issues/10 and similar problems that were found while coding the webvr-recoder. Since some properties in VRDisplay are read-only, everything needs to be wrapped and it's a big rewrite. Same with Gamepad API.

Adding VRFrameData is trivial, I'm just holding on to the spec changes while preparing the wrapper rewrite.

**Comment by AVGP** [comment]
Oh right, I see.
Just FYI: WebVR 1.1 has landed in Boris' polyfill directly: https://github.com/googlevr/webvr-polyfill/pull/149/files

So, yeah, let me know what I can do right now to help ya out! :wave:

**Comment by spite** [comment]
VRFrameData and VRDisplay.getFrameData implemented in https://github.com/spite/WebVR-Extension/commit/100da95e341eae326300c882d8f493d764d41e9a

Seems to work ok, but there's some weird issue with webvr.info/samples and Canary at the moment. All other content in Stable seems to work.
